test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeGenerator.java

```diff
@@ -1120,4 +1134,63 @@ public static void generateNonInterfaceClassWithMissingFlags(String name) {
 		byte[] bytes = classWriter.toByteArray();
 		generator.defineClass(name, bytes, 0, bytes.length);
 	}
+
+	public static void generateTestFieldCantHaveAccFinalAndAccVolatile() {
+		generateAccStaticTestGeneric("TestFieldCantHaveAccFinalAndAccVolatile", ACC_FINAL | ACC_VOLATILE, true);
+	}
+
+	public static void generateTestFieldCantHaveAccStrictAndAccStatic() {
+		generateAccStaticTestGeneric("TestFieldCantHaveAccStrictAndAccStatic", ACC_STRICT | ACC_STATIC, true);
+	}
+
+	public static void generateTestAccStrictFieldMustHaveAccFinal() {
+		generateAccStaticTestGeneric("TestAccStrictFieldMustHaveAccFinal", ACC_STRICT, true);
+	}
+
+	public static void generateTestValueClassFieldMustHaveAccStaticOrAccStrict() {
+		generateAccStaticTestGeneric("TestValueClassFieldMustHaveAccStaticOrAccStrict", 0, false);
+	}
+
+	public static Class<?> generateTestValueClassPutStrictFieldAfterInitialization() {
+		return generateAccStaticTestGeneric("TestAccStrictFieldMustHaveAccFinal", ACC_FINAL | ACC_STRICT, false, true);
```

Comment:
The test method name is "generateTestValueClassPutStrictFieldAfterInitialization", but the class name here is "TestAccStrictFieldMustHaveAccFinal".